keps/sig-api-machinery/3488-cel-admission-control/README.md

```diff
@@ -484,26 +493,158 @@ while another might validate the containers declared in the `podTemplate` of a
 is being enforced, just in different ways, it is reasonable for both to share
 the same policy configuration resources.
 
-Alternative considered: Embed the configuration CRD schema directly into
-`ValidatingAdmissionPolicy`.
+Alternative: `/matchRules` subresource
+
+```yaml
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: replicalimit.example.com
+spec:
+  group: example.com
+  versions:
+    ... # configuration schema(s) go here
+  subresources:
+    matchRules:
+      matchRulesPath: .spec.match
+```
 
 Pros:
 
-- Policy author doesn't need to define the CRD. It can instead be generated for them.
+- Follows pattern used by scale subresource to provide polymorphism across CRDs
```

Comment:
IIRC subresources like /scale and /status are also intended to provide the ability to access/modify that subresource independently. I don't think those capabilities are a requirement here. I don't suppose there's any way to use that to streamline things?

Would shoehorning a subresource-like thing into a CRD just for the sake of injecting a schema and ignoring the rest of the features fragment the notion of subresources?

Are there other KEPs that could leverage "subresource-lite"?

Reply:
I agree that this a very different usage of subresource than the existing kubernetes subresources.

The closest thing to "subresource-lite" might be `Content-Type: PartialObjectMetadata` which is a way to read only the metdata of kubernetes resources.  Note that there is no subresource for metadata. `PartialObjectMetadata` is unusual in that it provides access to a part of resources that already has a well defined structure, so it is not a particularly good match for what we're trying to do either.

I'll list what you mentioned here in cons.